### DISCUSSION TITLE
small paleontology portal

### DISCUSSION THREAD

**Discussion by seltmann** [Symbiota/community-board#149] [General]
Hi all-
UCSB has a small paleontology collection. Is there a portal we can get involved in to digitize and track these specimens? Thanks!! Katja

**Comment by themerekat** [comment]
Thanks for your thoughts! We've recently had a few similar inquiries, so we're definitely thinking about it. Stay tuned!

**Comment by ljwalker** [comment]
@seltmann This portal is still being set up (and will be throughout 2024), but, FYI: https://paleodata.biokic.asu.edu/portal/
